lts.json

```diff
@@ -2,13 +2,13 @@
   "autoprefixer": "~0.6.0",
   "broccoli-asset-rev": "^2.4.2",
   "dockerlint": "^1.0.0",
-  "ember-browserify": "~1.1.0",
+  "ember-browserify": "~1.1.13",
   "ember-can": "~0.8.0",
   "ember-cli": "~2.8.0",
-  "ember-cli-app-version": "^1.0.0",
+  "ember-cli-app-version": "^2.0.0",
   "ember-cli-babel": "^5.1.6",
   "ember-cli-clipboard": "~0.4.0",
```

Comment:
@sandersky is this something we should include as part of an endorsed/pinned set of addons?